projects/plugins/jetpack/_inc/client/components/jetpack-notices/style.scss

```diff
@@ -17,3 +17,9 @@
 	margin-right: 1rem;
 	line-height: 0;
 }
+
+.loading-notice {
+	height: 47px;
```

Comment:
It seems that we have two tabs here.

Reply:
@rcanepa The lines below this have spaces instead of tabs, fixed it now - you can see the [file here](https://github.com/Automattic/jetpack/blob/6a9002a44b5c75619cd2a7bdcdec7ac474018dd8/projects/plugins/jetpack/_inc/client/components/jetpack-notices/style.scss).Thanks